devcontainer.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This file needs to be in `.devcontainer/devcontainer.json`. See https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers